### ISSUE TITLE
[Recipe] Stargate parts not using Dimensional Alloy

### ISSUE THREAD

**Issue by Timeslice42** [JasonMcRay/InfiTech-1.7#865]
The SG Craft parts are still using the GT naquadah alloy. These recipes need to be changed to use the dimensional alloy.

P.S. Good job on the rendering effects for the dimensional alloy :)

**Comment by Nex4rius** [comment]
Why should they be changed to dimensional alloy?

**Comment by Timeslice42** [comment]
Because dimensional alloy is what naquadah alloy used to be when this recipe was created. GT5u later changed the naquadah alloy recipe to something vastly more expensive,

Dimensional alloy was literally added to the pack to replace where we had naquadah alloy in our custom recipes.

**Comment by JasonMcRay** [comment]
Stargate is build from Naquadah as per TV Show canon. It is how it is meant to be.

**Comment by Timeslice42** [comment]
Dimensional alloy *is* naquadah.

**Comment by Nex4rius** [comment]
1) dimensional alloy isnt called naquadah 
2) dimensional alloy uses enderium in the recipe which was never used before to make a stargate
3) naquadah alloy is naquadah ... why change it?

**Comment by Timeslice42** [comment]
1. A rose by any other name would still smell as sweet. I don't care what we call dimensional alloy; call it Empowered Naquadah... or whatever.
2. Naquadah alloy uses osmium and iridium too. When was that ever used to make a stargate? (Hint: Never.)
3. The stargate's usefulness was already marginalized back when it was an HV recipe, due to needing to find half the parts as dungeon loot. Making it an IV recipe is just silly.
4. The only reason I bothered creating this ticket is that this was already discussed with Jason, who agreed with the change, back when we designed dimensionial alloy. Jason either forgot, changed his mind, or just plain misunderstood. I don't have any plans to ever make a stargate, and if I ever do, I can change the recipe to be whatever I want.
